code/game/objects/buckling.dm

```diff
@@ -67,7 +67,7 @@
 
 //Wrapper procs that handle sanity and user feedback
 /obj/proc/user_buckle_mob(mob/living/M, mob/user)
-	if(!user.Adjacent(M) || user.lying || user.incapacitated())
+	if(!user.Adjacent(M) || user.lying || user.incapacitated() || user.stat)
```

Comment:
maybe you could change the incapacitated proc instead, since it checks for stat for living mobs. mob/proc/incapacitated currently doesn't do anything because I didn't think it would ever be used when I created it.